connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointTask.java

```diff
@@ -81,7 +93,14 @@ public void start(Map<String, String> props) {
         adminTimeout = config.adminTimeout();
         offsetSyncStore = new OffsetSyncStore(config);
         sourceAdminClient = AdminClient.create(config.sourceAdminConfig());
+        targetAdminClient = (KafkaAdminClient) Admin.create(config.targetAdminConfig());
         metrics = config.metrics();
+        scheduler = new Scheduler(MirrorCheckpointTask.class, config.adminTimeout());
```

Comment:
Looks like you forgot to shutdown the scheduler in close()

Reply:
done